plugin/evm/limit_order.go

```diff
@@ -32,12 +32,24 @@ const (
 	snapshotInterval    uint64 = 10 // save snapshot every 1000 blocks
 )
 
+type NodeType string
+
+const (
+	Tresor          NodeType = "tresor"          // vanilla validator
+	Kitkat          NodeType = "kitkat"          // matching engine
```

Comment:
[question] is it fair to say that kitkat type nodes are validators as well as matching engine? if yes, would it make sense to update the comment to say `validator + matching engine` to be more explicit?

Reply:
correct. edited.